ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/wildfly/extension/rts/main/module.xml

```diff
@@ -44,6 +44,7 @@
         <module name="org.jboss.resteasy.resteasy-jaxb-provider" services="export"/>
         <module name="org.hibernate.validator" services="export"/>
         <module name="org.jboss.narayana.rts"/>
+        <module name="org.jboss.narayana.lra"/>
```

Comment:
@marcosgopen are you sure RTS depends on LRA? I am a little surprised to see this as rts is for REST-AT?

Reply:
I removed it and tested it, it works. Thanks